pandas/core/arrays/categorical.py

```diff
@@ -1847,6 +1847,31 @@ def __iter__(self):
         """Returns an Iterator over the values of this Categorical."""
         return iter(self.get_values().tolist())
 
+    def __contains__(self, key):
+        """Returns True if `key` is in this Categorical."""
+        hash(key)
```

Comment:
isn’t this very similar to what u did in CI ?
can we unify

Reply:
I've made a unified version.

Reply:
so it looks very similiar. why cannot you just use ``key in self._data`` for the CI